From 2c4bd9fa876c26f5e694df62dc7c14d9643f53c5 Mon Sep 17 00:00:00 2001
From: tx-spp <yamashita.hideyuki@po.ntt-tx.co.jp>
Date: Mon, 6 Jan 2020 10:43:15 +0900
Subject: [PATCH 03/11] docs: change primary process startup parameter

This patch adds change on primary process startup command
when using offload with MLX5 supported NIC.

Signed-off-by: Hideyuki Yamashita <yamashita.hideyuki@ntt-tx.co.jp>
Signed-off-by: Naoki Takada <ntakada14@gmail.com>
---
 docs/guides/gsg/howto_use.rst | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/docs/guides/gsg/howto_use.rst b/docs/guides/gsg/howto_use.rst
index b0824d4..6d84be7 100644
--- a/docs/guides/gsg/howto_use.rst
+++ b/docs/guides/gsg/howto_use.rst
@@ -415,6 +415,26 @@ secondary processes.
         -n 10 \
         -s 192.168.1.100:5555
 
+In case of using MLX5 supported NIC, you must add ``dv_flow_en=1``
+with white list option.
+
+.. code-block:: console
+
+    # terminal 3
+    $ sudo ./src/primary/x86_64-native-linuxapp-gcc/spp_primary \
+        -l 0 -n 4 \
+        -w 0000:03:00.0,dv_flow_en=1 \
+        -w 0000:04:00.0,dv_flow_en=1 \
+        -w 0000:05:00.0 \
+        --socket-mem 512,512 \
+        --huge-dir /dev/hugepages \
+        --proc-type primary \
+        --base-virtaddr 0x100000000
+        -- \
+        -p 0x03 \
+        -n 10 \
+        -s 192.168.1.100:5555
+
 - EAL options:
 
   - ``-l``: core list
-- 
2.17.1

